src/main/kotlin/racingcar/domain/Setting.kt

```diff
@@ -0,0 +1,20 @@
+package racingcar.domain
+
+const val DEFAULT_LAB_COUNT = 5
```

Comment:
```suggestion
private const val DEFAULT_LAB_COUNT = 5
```
DEFAULT_LAB_COUNT는 Setting 클래스에서만 접근하는 상수값이네요.
그렇다면 외부에서는 접근하지 못하도록, private 접근자를 붙여보면 어떨까요?
추가로 top-level 변수로 만드셔도 좋지만, companion object에 정의하셔도 좋습니다. 😄